ui/app/pages/add-token/add-token.container.js

```diff
@@ -14,7 +14,7 @@ const mapStateToProps = (state) => {
     mostRecentOverviewPage: getMostRecentOverviewPage(state),
     tokens,
     pendingTokens,
-    showSearchTab: getIsMainOrTestNet(state),
+    showSearchTab: getIsMainnet(state) || process.env.IN_TEST,
```

Comment:
Nit: It might be better to check explicitly against the string `true` here. Though of course `true` is truthy, and this isn't set at all outside of tests, so this still gives the correct result.

```suggestion
    showSearchTab: getIsMainnet(state) || process.env.IN_TEST === 'true',
```